docs/SOP.md

```diff
@@ -100,6 +101,17 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+## Reviving obsolete, orphaned, or inactive ontologies
+Ontologies that became dormant and were marked as either obsolete, orphaned, or inactive require a review process if they want to be marked as active again.
+Currently, these are the required steps:
+1. If an ontology is desired to be reactivated, a member of the ontology community first makes a PR on the metadata file with the desired change in state.
+This PR must include some evidence that backs up the change (we will provide better guidelines as this SOP matures). Think of showing closed issues to indicate activity etc.
+2. If the community member is not the ontology owner, the ontology owner must be contacted for comment via email and tagged on the issue. The ontology owner is given 1 month to object to the change, in which case, it is rejected.
```

Comment:
A lot of this SOP is different for orphaned-> active and inactive->active, it might make sense to split these